### ISSUE TITLE
[MacOS] Backtick 

### ISSUE THREAD

**Issue by rbaldini** [Zettlr/Zettlr#5517]
### Description

While writing a text, pressing `Option` + `|` doesn't return backtick ( \` ).
This keyboard sequence is working fine in any text editor, word processor, text field and so on.
It worked fine in Zettlr till version 3.2.1; with version 3.3.1, pressing `Option` + `|` seems to be unrecognized and nothing is outputted.

### Reproducing

Open a document in Zettlr v. 3.3.1 and press `Option` + `|` . Nothing will happen.

### Zettlr version

3.3.1

### Installation Method

From the Website or GitHub

### Your Platform

- [ ] Windows
- [X] macOS
- [ ] Linux

### Architecture

- [ ] Intel 64bit
- [X] ARM 64bit (includes Apple Silicon)

### Operating System Version

Sonoma 14.7.1 and Sequoia 15.1.1

### Additional Information

Tested both on Mac OS Sonoma 14.7.1 and Sequoia 15.1.1, with both architecture M1 and M3.
I've tried installing both from Website and Homebrew: the issue is the same.

**Comment by nathanlesage** [comment]
Please provide your keyboard layout, as this issue relates to how your keyboard maps characters.

**Comment by rbaldini** [comment]
It's an italian keyboard. with the old Zettlr versions worked perfectly.
![Keyb](https://github.com/user-attachments/assets/61b84b75-98ce-4dfd-8221-1428b8019fb0)

**Comment by nathanlesage** [comment]
Thanks, I'll have a try. It's likely a keyboard shortcut that interferes there. Does pressing `Opt-E` and then a space produce a backtick? That could be a quick fix until we've figured this out.

**Comment by rbaldini** [comment]
Thanks Hendrik, `Opt-E` in my keyboard layout is immediately delivering `€` without the need to press `space`. You can see the `€` on the `E` key.
As told, also here adding this comment, pressing `Opt-|` is devevering backtick.

**Comment by rbaldini** [comment]
I did some test... with Italian keyboard, in Zettlr, to leverage the backtick "accent" I have to go for `Opt-9` but I would like it back under `Opt-Pipe`.
Is there any Zettlr config file I could update?

**Comment by nathanlesage** [comment]
I now switched to the macOS `it-IT` keyboard layout that you're using and I can confirm that inserting a backtick without deadkey on `Opt+|` works on that keyboard layout with all text fields — including in Zettlr — except the main editor. I'm going to investigate, since it's likely interference from any of the extensions in the editor.

**Comment by FedericoMassaioli** [comment]
Hi, any news about this issue?

**Comment by nathanlesage** [comment]
No. Feel free to investigate and open a PR if the workaround doesn't work for you. I'll happily merge such a PR.
